bears/general/CLOCBear.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='cloc')
+class CLOCBear:
+    """
+    Summarises a file or number of files in directory structure
+    with total lines, number of comment lines, number of actual code lines
+    using CLI tool cloc
+    """
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='cloc')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Number of comment lines',
+                  'Number of blank lines',
+                  'Number of code lines'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return ('--csv', filename)
+
+    def process_output(self, output, filename, file):
+        lines = output.split('\n')
+        files, ignored_files = re.search(
+            r'(\d+).*?\n\s+(\d+)', '\n'.join(lines[1:3]), re.MULTILINE)\
+            .group(1, 2)
+        if(files == ignored_files):
+            msg = 'File does not belong to valid programming language.'
+            yield Result.from_values(origin=self,
+                                     message=msg,
+                                     file=filename,
+                                     severity=RESULT_SEVERITY.INFO)
```

Comment:
@jayvdb I had initially kept `severity=RESULT_SEVERITY.MAJOR`. But @SanketDG disapproved it and asked to change to INFO. According to me, it should be MAJOR or at least NORMAL.
I'm pushing all the changes as requested with severity MAJOR. Let me know if it needs to be NORMAL.